Distance.java

```diff
@@ -0,0 +1,101 @@
+import java.util.Objects;
+
+public class Distance {
+	/* Napisz klase ktora trzyma informacje o wartosci odleglosci i
+    o rodzaju miary METRY KILOMETRY CENTYMETRY w klasie ma byc metoda ktora
+    porownuje dwa miary medota toString ktora ladnie wypsuize miare np. 10 km i
+    metoda ktora przyjmuje typ miary i zmieni typ miary obiektu na inny
+    *///wtf
+
+
+	enum DistanceUnitType
+    {
+        CENTIMETERS,
+        METERS,
+        KILOMETERS
+    }
+
+    private double distance;
+    private DistanceUnitType measure;
+    private double cmMeasure;
+
+    Distance(double distance, DistanceUnitType measure)
+    {
+        this.distance = distance;
+        this.measure = measure;
+
+        if(Objects.equals(measure, DistanceUnitType.KILOMETERS))
+        {
+            cmMeasure = this.distance *100000;

```

Comment:
instead of magic number 100000 define constant named like CENTIMETERS_IN_KILOMETER

Reply:
do that for every 'magic number'
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad